### DISCUSSION TITLE
Improve image sharpness

### DISCUSSION THREAD

**Discussion by ptilou** [ImageMagick/ImageMagick#7036] [Help]
### ImageMagick version

7.0.0.0

### Operating system, version and so on

Fedora 38

### Description

Hi,

I cut the movie, by ffmpeg.
I want how to sharpness by imagemagick this picture : 
![0086](https://github.com/ImageMagick/ImageMagick/assets/7466469/f2b5b191-473e-4c2c-874e-f8ae7a7dd437)

Great

Philippe

**Comment by fmw42** [comment]
ffmpeg has deconvolution filters.  Use them.  Imagemagick does not have deconvolution capabilities.

**Comment by ptilou** [comment]
Hello,

I'm testing : https://ffmpeg.lav.io/ , from deconvolution, i'm don't good return !
My be you've more solution ?

Philippe

**Reply by fmw42** [reply]
Imagemagick does not have deconvolution capabilities

**Reply by ptilou** [reply]
Hello,

> Imagemagick does not have deconvolution capabilities

What' : http://www.fmwconcepts.com/imagemagick/fftdeconvol/index.php
It's your !
I wanted how to make contact photo, i've that's, it's enjoy, but do you solution from loop and processing imagemagick just or and more software ?

Philippe

**Reply by fmw42** [reply]
This is my bash shell script for Imagemagick.  You can try it.  But to know how to use it better. Read and study http://www.fmwconcepts.com/imagemagick/fourier_transforms/fourier.html.  This script an my others work only under special circumstances and photos and generally do not work well on real-life photos.  See the cepstrum script to try to figure out what kind of "blur" you have in your photo.
